### PR TITLE
trafficserver 5.2.0

### PR DESCRIPTION
**Pull request by felixbuenemann**

### CODE REVIEW

File: Library/Formula/trafficserver.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Trafficserver < Formula
   homepage "http://trafficserver.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=trafficserver/trafficserver-5.1.2.tar.bz2"
-  mirror "http://archive.apache.org/dist/trafficserver/trafficserver-5.1.2.tar.bz2"
-  sha1 "907edeebd3a10d4787ad550e735ea196f03deb04"
+  url "http://www.apache.org/dyn/closer.cgi?path=trafficserver/trafficserver-5.2.0.tar.bz2"
```
Comment (asparagui) [inline Library/Formula/trafficserver.rb]: Please switch this to https to pass audit.

**Comment by asparagui** [comment]
@BrewTestBot test this please

**Comment by felixbuenemann** [comment]
Btw, both 5.1.2 and 5.2.0 fail to compile with clang's libc++  from Xcode 6.3 beta 3, while HEAD builds. 
I'll see if I can come up with a patch (ENV.libstdcxx also works). Xcode 6.2 is not affected.

**Comment by felixbuenemann** [comment]
@asparagui The url point to a page with mirrors, so it's not a direct download url. Should I remove it and just use the mirror url?

**Comment by felixbuenemann** [comment]
I'll try reverting that url back to http, brew bot only complained about https on the mirror url.

**Comment by felixbuenemann** [comment]
The problem is:
Library/Homebrew/download_strategy.rb:824:    when %r[^http://www\.apache\.org/dyn/closer\.cgi]
it should probably match https as well.

**Comment by mikemcquaid** [comment]
Thanks again @felixbuenemann!

**Pull request closed without merging**